plugins/scheduled_task/lib/processors/KMediaRepurposingProcessor.php

```diff
@@ -122,6 +129,36 @@ private function getMetadataOnObject($objectId, $metadataProfileId)
 		return null;
 	}
 
+	private function createProcessField($object, $task)
+	{
+		$distributeTaskType = ScheduledTaskContentDistributionPlugin::getApiValue('Distribute');
```

Comment:
enum?

Reply:
Interface 'IKalturaPluginEnum' not found